


#global testerror "Err0511"
func(op: string, op: s32) toto()
{
}
